src/features/NoteEditor/NoteVersions.tsx

```diff
@@ -145,8 +158,13 @@ export const NoteVersions = ({
 								</HStack>
 								<HStack marginLeft="auto">
 									<Button
+										isDisabled={Boolean(isReadOnly)}
 										size="sm"
-										title="Apply version"
+										title={
+											isReadOnly
+												? 'Not available for for read-only notes'
```

Comment:
Not available what?

The `title` purpose is to explain a context. Here you explain why button is disabled, but do not explain what this button is.

We should never cut the context and only extend when it possible.
For example `Version cannot be applied for a read-only note`